code/game/machinery/cryopod.dm

```diff
@@ -97,7 +97,7 @@
 		//dat += "<a href='?src=\ref[src];item=1'>Recover object</a>.<br>" //VOREStation Removal - Just log them.
 		//dat += "<a href='?src=\ref[src];allitems=1'>Recover all objects</a>.<br>" //VOREStation Removal
 
-	to_chat(user, browse(dat, "window=cryopod_console"))
+	user << browse(dat, "window=cryopod_console")
```

Comment:
...Is this supposed to be going back to user <<?

Reply:
Well, this isn't a chat situation, so it makes sense NOT to use to_chat there.